src/Rings/msolve/msolve.jl

```diff
@@ -0,0 +1,180 @@
+import msolve_jll: libmsolve
+
+export msolve
+
+@doc Markdown.doc"""
+    function get_rational_parametrization(nr::Int32, lens::Array{Int32,1}, cfs::Ptr{BigInt})
+
+Construct the rational parametrization of the solution set computed via msolve.
+
+**Note**: This is an internal function and should only be used inside `msolve()`.
+"""
+function get_rational_parametrization(
+        nr::Int32,
+        lens::Vector{Int32},
+        cfs::Ptr{BigInt}
+    )
+    C, x  = PolynomialRing(QQ,"x")
+    ctr   = 0
+
+    elim  = C([unsafe_load(cfs, i) for i in 1:lens[1]])
```

Comment:
I didn't try it, but I think you could probably use `Base.unsafe_wrap(Array, ...)` and below (as you do elsewhere) for better performance / less memory usage (although I suspect it doesn't matter here :-) -- so feel free to ignore). I.e. something like this (untested!)
```suggestion
    elim  = C(Base.unsafe_wrap(Array, cfs, lens[1]))
```

Reply:
I tried this, but I am getting segfaults then. Maybe we leave this for further improvements later on when I get more functionality from the msolve library to Oscar. From the performance POV this step is negligible.